Control/Monad/Combinators.hs

```diff
@@ -176,6 +179,25 @@ manyTill p end = go id
           go (f . (x:))
 {-# INLINE manyTill #-}
 
+-- | @'manyTill_' p end@ applies parser @p@ /zero/ or more times until parser
+-- @end@ succeeds. Returns the list of values returned by @p@ and the @end@ result.
+--
+-- Use @manyTill@ if you have no need in the result of the @end@.
+--
+-- See also: 'skipMany', 'skipManyTill'.
```

Comment:
You could add `@since 1.2.0` here. Also note that once we have `manyTill_` the old `manyTill` can be defined in terms of it.

Reply:
Added and defined.